exo-create/features/steps/steps-when.ls

```diff
@@ -10,7 +10,7 @@ require! {
 
 module.exports = ->
 
-  @When /^executing "([^"]*)"$/ (command, done) ->
+  @When /^executing "([^"]*)"$/, timeout: 20_000, (command, done) ->
```

Comment:
Can you investigate this using debug statements (adding a bunch of `console.log` statements to see which step takes a long time)?